watttime/api.py

```diff
@@ -690,3 +691,63 @@ def get_maps_json(
         url = "{}/v3/maps".format(self.url_base)
         params = {"signal_type": signal_type}
         return self._make_rate_limited_request(url, params)
+
+
+class WattTimeMarginalFuelMix(WattTimeBase):
+
+    def get_fuel_mix_jsons(
+        self,
+        start: Union[str, datetime],
+        end: Union[str, datetime],
+        region: str,
+        signal_type: Optional[Literal["marginal_fuel_mix"]] = "marginal_fuel_mix",
+        model: Optional[Union[str, date]] = None,
+    ) -> List[Dict[str, Any]]:
+        if not self._is_token_valid():
+            self._login()
+        url = f"{self.url_base}/v3/fuel-mix"
+        responses = []
+        params = {
+            "region": region,
+            "signal_type": signal_type,
+        }
+
+        start, end = self._parse_dates(start, end)
+        chunks = self._get_chunks(start, end, chunk_size=timedelta(days=30))
+
+        # No model will default to the most recent model version available
+        if model is not None:
```

Comment:
```suggestion
        if model:
```
to also ignore `model=""`